wordpress-mobile/gutenberg-mobile
# Standardizes the post title editing logic.

## Description:

Standardizes the mechanism for editing the post title.

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/512

## Testing:

1. Edit the title
2. Switch to HTML mode and back to visual.

Make sure the title is still as modified.
